modules/custom/og_ext_cron/src/Utils/CronFunctions.php

```diff
@@ -524,5 +524,214 @@ public function generate_vote_count_json_file()
+        $ownerOrgCode = $this->get_item_interface_field_value( $_atiIndexItem, 'org_name_code' );
+        $ownerOrgNameEn = $this->get_item_interface_field_value( $_atiIndexItem, 'org_name_en' );
+        $ownerOrgNameFr = $this->get_item_interface_field_value( $_atiIndexItem, 'org_name_fr' );
+
+        if(
+          is_null($id) ||
+          is_null($requestNumber) ||
+          is_null($summaryEn) ||
+          is_null($summaryFr) ||
+          is_null($ownerOrgCode) ||
+          is_null($ownerOrgNameEn) ||
+          is_null($ownerOrgNameFr)
+        ){
+
+          continue;
+
+        }
+
+        $parsedAtiIndexItems[$id] = [
+          'request_number'    => $requestNumber,
+          'summary_en'        => $summaryEn,
+          'summary_fr'        => $summaryFr,
+          'owner_org_code'    => $ownerOrgCode,
+          'owner_org_name_en' => $ownerOrgNameEn,
+          'owner_org_name_fr' => $ownerOrgNameFr
+        ];
+
+      }
+
+      $rows = [];
+      $missingIndexItemsCounter = 0;
+      foreach( $localAtiRequestCounts as $_id => $_years ){
+
+        if( ! array_key_exists( $_id, $parsedAtiIndexItems ) ){
+          $missingIndexItemsCounter++;
+          continue;
+        }
+
+        foreach( $_years as $_year => $_months ){
+
+          foreach( $_months as $_month => $_count ){
+
+            $rows[] = [
+              'year'              => $_year,
+              'month'             => $_month,
+              'id'                => $_id,
+              'request_number'    => $parsedAtiIndexItems[$_id]['request_number'],
+              'summary_en'        => $parsedAtiIndexItems[$_id]['summary_en'],
+              'summary_fr'        => $parsedAtiIndexItems[$_id]['summary_fr'],
+              'owner_org_code'    => $parsedAtiIndexItems[$_id]['owner_org_code'],
+              'owner_org_name_en' => $parsedAtiIndexItems[$_id]['owner_org_name_en'],
+              'owner_org_name_fr' => $parsedAtiIndexItems[$_id]['owner_org_name_fr'],
+              'request_count'     => $_count,
+            ];
+
+          }
+
+        }
+
+      }
+
+      if( $missingIndexItemsCounter > 0 ){
+        \Drupal::logger('cron')->notice("$missingIndexItemsCounter requests not matched. ATI Summaries not found in the core_ati index...");
+      }
+
+      $this->write_to_csv(
+        'ati-informal-requests-analytics.csv',
+        $rows,
+        [
+          'Year',
+          'Month',
+          'Unique Identifier',
+          'Request Number',
+          'Summary - EN',
+          'Summary - FR',
+          'owner_org',
+          'Organization Name - EN',
+          'Organization Name - FR',
+          'Number of Informal Requests'
+        ],
+        true
+      );
+      
+      $filePath = \Drupal::service('file_system')->realpath(\file_default_scheme() . "://") . '/ati-informal-requests-analytics.csv';
+      $ckanFilePath = '/opt/tbs/ckan/smb/portal/public/ati-informal-requests-analytics.csv';

```

Comment:
you can use `ckan_public_path` configuration in Drupal Settings to get file path

Reply:
@RabiaSajjad it should now be using `ckan_public_path` Drupal setting instead of the static string.